src/main/java/scene/clue/CheckedClueTrackerBuilder.java

```diff
@@ -1,20 +1,19 @@
 package scene.clue;
 
 import scene.suspect.SuspectList;
+
 import static scene.suspect.SuspectListBuilder.suspectListBuilder;
 
 import java.io.FileNotFoundException;
 
 public class CheckedClueTrackerBuilder {
-    /*private Ui ui;
-    private SuspectList suspects;
-
-    public SearchedClueTracker(Ui ui) {
-        this.ui = ui;
-        this.suspects = new SuspectList(ui);
-        buildSuspectList();
-    }*/
 
+    /** Builds a tracker for already checked clues according
+     *  to a text file that records relevant information.
+     *
```

Comment:
Can change the Bulids to next line to follow the javadocs format

Reply:
Thanks for the comments! have changed accordingly